airflow/www/templates/airflow/dags.html

```diff
@@ -60,9 +60,12 @@ <h2>DAGs</h2>
         </div>
         <div class="col-sm-4">
           <form id="search_form" class="form-inline">
-            <div class="form-group" style="width: 100%;">
+            <div class="form-group search-input" style="width: 100%;">
               <label for="dag_query" class="sr-only">Search DAGs</label>
-              <input type="search" id="dag_query" class="typeahead form-control" data-provide="typeahead" style="width:100%;" value="{{search_query}}" autocomplete="off" placeholder="Search DAGs">
+              <input type="search" id="dag_query" class="typeahead form-control search-input__input" data-provide="typeahead" style="width:100%;" value="{{search_query}}" autocomplete="off" placeholder="Search DAGs">
+              {% if search_query %}
+                <button type="reset" class="btn btn-default btn-sm material-icons search-input__clear-btn">cancel</button>
```

Comment:
```suggestion
                <button type="reset" class="btn btn-default btn-sm material-icons search-input__clear-btn">Clear DAG Search Term</button>
```

Might make it better if someone tries using a TTS browser?

Reply:
"cancel" is the required sting for material icons to render the desired icon. I went ahead and added an `aria-label` attribute instead.